lib/MusicBrainz/Server/Controller/Role/Rating.pm

```diff
@@ -1,19 +1,34 @@
 package MusicBrainz::Server::Controller::Role::Rating;
 use Moose::Role -traits => 'MooseX::MethodAttributes::Role::Meta::Role';
 
+use MusicBrainz::Server::Constants qw( %ENTITIES );
+
 requires 'load';
 
 sub ratings : Chained('load') PathPart('ratings')
 {
     my ($self, $c) = @_;
 
     my $entity = $c->stash->{$self->{entity_name}};
-    my @ratings = $c->model($self->{model})->rating->find_by_entity_id($entity->id);
-    $c->model('Editor')->load(@ratings);
-    $c->model('Editor')->load_preferences(map { $_->editor } @ratings);
+    my $entity_properties = $ENTITIES{ $entity->entity_type // 0 };
```

Comment:
Do you remember why/when `$entity->entity_type` is undef there?

Reply:
It might never be, I might have just copied this as-is from elsewhere, can't remember rn. Guess we can remove it and see if it breaks.